components/gitpod-db/.vscode/launch.json

```diff
@@ -5,7 +5,8 @@
             "type": "node",
             "request": "launch",
             "env": {
-                "DB_PORT": "23306"
+                "DB_PORT": "23306",
+                "DB_ENCRYPTION_KEYS": "[{\"name\":\"general\",\"version\":1,\"primary\":true,\"material\":\"5vRrp0H4oRgdkPnX1qQcS54Q0xggr6iyho42IQ1rO+c=\"}]"
```

Comment:
is it safe to include it here? it's the "Test DB", right?

Reply:
yes. It's the same we have in the leeway script to initialize the env file